src/Aspire.Dashboard/Model/DashboardCommandExecutor.cs

```diff
@@ -91,7 +92,7 @@ public async Task ExecuteAsync(ResourceViewModel resource, CommandViewModel comm
             toastParameters.Icon = GetIntentIcon(ToastIntent.Error);
             toastParameters.Content.Details = response.ErrorMessage;
             toastParameters.PrimaryAction = loc[nameof(Dashboard.Resources.Resources.ResourceCommandToastViewLogs)];
-            toastParameters.OnPrimaryAction = EventCallback.Factory.Create<ToastResult>(this, () => navigationManager.NavigateTo(DashboardUrls.ConsoleLogsUrl(resource: resource.Name)));
+            toastParameters.OnPrimaryAction = EventCallback.Factory.Create<ToastResult>(this, () => navigationManager.NavigateTo(DashboardUrls.ConsoleLogsUrl(resource: resource.GetConsoleLogsName(resources))));
```

Comment:
I think `Func<ResourceViewModel, string> getResourceName` on this method can already do this.